src/main/programming/logic_condition.c

```diff
@@ -333,6 +334,15 @@ static int logicConditionCompute(
             }
             break;
 
+        case LOGIC_CONDITION_SET_PID_PROFILE:
+            operandA--;
+            if ( getConfigProfile() != operandA  && (operandA >= 0 && operandA < MAX_PROFILE_COUNT)) {
+                return setConfigProfile(operandA);
```

Comment:
This is not enough. I mean, yes, it activates the profile, but does not switches PID controller to the new one. See for example `pidInitFilters` and `pidInit`

Reply:
I based this on the CMS. But I guess that does a save and reboot, so none of that is necessary. I'll get it updated.